src/cli.js

```diff
@@ -10,12 +10,17 @@ const loadCommonConfig = require('./load-common-config');
 const mergeConfigs = require('./merge-configs');
 const wrilteFile = require('./write-file');
 
+
+function items(val) {
+  return val.split(",");
+}
+
 module.exports.exec = () => {
   commander
   .version('0.0.15')
   .description('Import functions from swagger spec filet to serverless.yml')
   // Common options.
-  .option('-i, --input <path>', 'Specify swagger file path. (defailt "./swagger.ya?ml")')
+  .option('-i, --input <path>', 'Specify swagger file path. (defailt "./swagger.ya?ml")', items)
```

Comment:
How about `-i foo.yaml -i bar.yaml` style instead of `-i foo.yaml,bar.yaml`? It seems more common for multiple path arguments.
 
```
function collect(val, memo) {
  memo.push(val);
  return memo;
}

...

commander
.option('-c, --collect [value]', 'A repeatable value', collect, [])
```
https://github.com/tj/commander.js/blob/v2.19.0/Readme.md#coercion

Reply:
I don't have any evidence regarding the poupalirty between styles of multiple arguments.
Actually I have no prefernce on the styling, so I leave the desigin decision to @AKIRA-MIYAKE 's choice.

Reply:
Well, I will update PR following @unarist's suggestion.
I am afraid  a situation when the paths contains comma(,)

Reply:
Done in 
https://github.com/AKIRA-MIYAKE/serverless-import-swagger/pull/27/commits/44b515462f21ad7284723c4b3a96e0187691d0ee